packages/ui-components/src/components/Map/AreaTooltip.js

```diff
@@ -81,7 +37,23 @@ export const AreaTooltip = ({
       permanent={permanent}
       interactive={permanent}
     >
-      <div style={{ display: 'grid' }}>{displayText}</div>
+      {text ? (
+        <Grid>{text}</Grid>
```

Comment:
This text prop is only used in lesmis. I think it would be better to refactor lesmis to use a simpler Tooltip component but I will do that in another PR after the release.